RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -223,6 +223,148 @@ int pullsNeeded(string data)
 	return count;
 }
 
+void bedtime_pulls()
+{
+	if(pulls_remaining() < 1)		//out of pulls or in hardcore or in casual.
+	{
+		return;
+	}
+	if(get_property("auto_bedtime_pulls_skip").to_boolean())
+	{
+		return;
+	}
+	
+	if(item_amount($item[Muculent Machete]) == 0 && my_class() != $class[Avatar of Boris])
+	{
+		pullXWhenHaveY($item[Antique Machete], 1, 0);
+	}
+	if(item_amount($item[Wet Stunt Nut Stew]) == 0 && !possessEquipment($item[Mega Gem]))
+	{
+		pullXWhenHaveY($item[wet stew], 1, 0);
+	}
+	if(!black_market_available())
+	{
+		pullXWhenHaveY($item[blackberry galoshes], 1, 0);
+	}
+	if(internalQuestStatus("questL11Desert") < 1)
+	{
+		pullXWhenHaveY($item[drum machine], 1, 0);
+		pullXWhenHaveY($item[killing jar], 1, 0);
+	}
+	
```

Comment:
Other suggestions of quest specific pulls to add here would be

- Spooky-Gro Fertilizer when `!hidden_temple_unlocked()` and `!in_glover()`
- harem veil and pants when the L5 quest is still not complete
- ores & ninja gear when the L8 quest is still not complete
- up to 4 rusty hedge trimmers based on the value of twinPeakProgress (see https://wiki.kolmafia.us/index.php/Quest_Tracking_Preferences#twinPeakProgress)
- Amulet of Extreme Plot Significance & Mohawk Wig when the L10 quest is still not complete
- lynyrdskin breeches, lynyrdskin cap, lynyrdskin tunic & lynyrd musk when we have 4 or more pulls remaining and the zeppelin mob hasn't been started (`get_property("zeppelinProtestors").to_int() == 0`)
- up to 4 Shore Inc. Ship Trip Scrip. 3 for island access if required, 1 for the UV compass if required
- war outfit if required.

I'm not saying you have to add these now if you want to get this merged sooner rather than later. This is just a suggestion for future additions.

Reply:
thank you. Yea I think they should go in a followup update. I even already have a followup update